### ISSUE TITLE
Documentation build is failing

### ISSUE THREAD

**Issue by cticenhour** [idaholab/TMAP8#40]
Documentation build is currently failing on CIVET: https://civet.inl.gov/job/1270817/

Seems to be due to the merge of idaholab/moose#22712. TMAP8 is missing syntax related to Thermochimica. Tagging @dschwen for awareness. Not sure when I will be able to get to this - perhaps tomorrow.

**Comment by FuerstT** [comment]
@cticenhour is this for documentation on https://mooseframework.inl.gov/TMAP8/syntax/tmap_only.html? 

I noticed some missing features, particularly the surface reaction boundary condition (recombination and dissociation). Or is my memory just failing me?

**Comment by cticenhour** [comment]
No, this particular failure is due to a syntax configuration issue more than likely, which is affecting the entire site build. 

The BCs you mentioned are documented on https://mooseframework.inl.gov/TMAP8/syntax/ which shows all available objects for a TMAP8 input and not only those housed in this repository (which is what the link you posted is showing). Many objects were moved to the MOOSE Scalar Transport Module as of #32. Via the link I shared, see the documentation for [BinaryRecombinationBC](https://mooseframework.inl.gov/TMAP8/source/bcs/BinaryRecombinationBC.html) and [DissociationBC](https://mooseframework.inl.gov/TMAP8/source/bcs/DissociationFluxBC.html). Hope this helps!
